api/handlers.go

```diff
@@ -289,13 +303,42 @@ func (a *API) Unmount(w http.ResponseWriter, r *http.Request) {
 
 	volName := volConfig.String()
 
+	ut := &config.UseMount{
+		Volume:   volName,
+		Reason:   lock.ReasonMount,
+		Hostname: a.Hostname,
+	}
+
+	if !volConfig.Unlocked {
+		a.RemoveStopChan(volName)
```

Comment:
Can you please explain me the use of `a.AddStopChan(volName, stopChan)` and `a.RemoveStopChan(volName)`. I couldn't get much from the code.